src/tests/article.test.js

```diff
@@ -203,4 +204,78 @@ describe('POST AND GET /api/v1/articles', () => {
         done();
       });
   });
+
+  // Rating Tests
+
+  describe('POST /api/v1/articles/{id}/rate', () => {
+    it('Should not be able to rate your own article', (done) => {
+      chai
+        .request(app)
+        .post(`/api/v1/articles/${articleId}/rate`)
+        .set('Authorization', userToken)
+        .send({ rate: 1 })
+        .end((err, res) => {
+          if (err) done(err);
+          expect(res).have.status(400);
+          expect(res).to.be.an('object');
+          expect(res.body.error).to.deep.equal('Sorry! You cannot rate your article');
+          done();
+        });
+    });
+    it('Should Login another user and return token', (done) => {
```

Comment:
this test doesn't belong here. There should be a test util that enables you to signup a new user and login as the new user

Reply:
on it. Thanks